doc/source/user_guide/mesh_geometry.rst

```diff
@@ -61,9 +90,202 @@ To view the current mesh status, you can use this code:
+|                        | **Default output**       | :py:`return_as_list=True`     | :py:`return_as_array=True`      | :py:`return_ids_in_array=True`      |
++========================+==========================+===============================+=================================+=====================================+
+| |get_keypoints|        | |pv-polydata|            | |list-pv-polydata|            | |np-array|                      | |np-array|                          |
++------------------------+--------------------------+-------------------------------+---------------------------------+-------------------------------------+
+| |get_lines|            | |pv-polydata|            | |list-pv-polydata|            | **Not Applicable**              | **Not Applicable**                  |
++------------------------+--------------------------+-------------------------------+---------------------------------+-------------------------------------+
+| |get_areas|            | |pv-polydata|            | |list-pv-unstruc|             | **Not Applicable**              | **Not Applicable**                  |
++------------------------+--------------------------+-------------------------------+---------------------------------+-------------------------------------+
+| |get_volumes|          | |pv-polydata|            | |list-pv-unstruc|             | **Not Applicable**              | **Not Applicable**                  |
++------------------------+--------------------------+-------------------------------+---------------------------------+-------------------------------------+
+
+.. _migration_geometry_api_066:
+
+Migration to the new Geometry API
+---------------------------------
+
+Several changes need to be addressed in older to update your script
+from the old API to the new API.
+
+One of the most important is that now, you don't need to call the entities like in the old API:
+
+.. code:: pycon
+
+   # Old API
+   >>> mapdl.geometry.areas()
+   [UnstructuredGrid (0x7f14add95040)
+     N Cells:	12
+     N Points:	20
+     X Bounds:	-2.000e+00, 2.000e+00
+     Y Bounds:	0.000e+00, 1.974e+00
+     Z Bounds:	0.000e+00, 0.000e+00
+     N Arrays:	4,
+   UnstructuredGrid (0x7f14add95ca0)
+     N Cells:	12
+     N Points:	20
+     X Bounds:	-2.000e+00, 2.000e+00
+     Y Bounds:	0.000e+00, 1.974e+00
+     Z Bounds:	5.500e-01, 5.500e-01
+     N Arrays:	4,
+   ...
+
+Whereas in the new API:
+
+.. code:: pycon
+
+   >>> mapdl.geometry.areas
+   MultiBlock (0x147ca7640)
+     N Blocks    28
+     X Bounds    -0.016, 0.016
+     Y Bounds    -0.008, 0.018
+     Z Bounds    -0.003, 0.015
+
+In addition, the type of entities returned by those methods are different.
+For comparison between the new and old geometry API, the following table is provided.
+
+.. _Table-2:
+
+**Table 2. Comparison between objects returned by both geometry APIs.**
```

Comment:
Should "geometry" here be capitalized? Is it referring to PyGeometry? I think within the topic, you should be consistent and "officially" define the "geometry API" in the first reference in the topic but then only use the "old API" and new "API". Sometimes throwing in "geometry" and sometimes leaving it out seems awkward.

Reply:
It is not referring to PyGeometry. I feel like having APIs "naked" is confusing. But yeah, I see your point.